components/popover/popover.jsx

```diff
@@ -509,19 +509,22 @@ class Popover extends React.Component {
 	};
 
 	render() {
+		const otherChildren = [];
```

Comment:
`Multiple children of any kind are allowed, but the first child must serve as the trigger component. ` should be documented in the children prop comment.

Current prop comment is:
`* The trigger of the component. This must be a **single node**. Many props will be passed into this trigger related to popover interactions. The child needs to be a clickable element, such as a `Button` or an anchor tag (`a`).`